js2r-helpers.el

```diff
@@ -25,6 +25,15 @@
 (require 'dash)
 (require 's)
 
+(defun js2r--wrap-text-with-prefered-quotes (&rest text)
+  "Wrap TEXT with the prefered quotes.  The prefered quotes is set with `js2r-prefered-quote-type'."
+  (let ((prefered-quotes "\""))
+    (when (= 2 js2r-prefered-quote-type)
+      (setq prefered-quotes "'"))
+    (concat prefered-quotes (apply 'concat text) prefered-quotes)))
+
+(defalias 'js2r--wrap-text 'js2r--wrap-text-with-prefered-quotes)
```

Comment:
This was because I found `js2r--wrap-text-with-prefered-quotes` to be too long to fit inside the `js2r--figure-out-what-to-log-where`. I'll definitely change it.